js/SplitButton/Easy-customization.md

```diff
@@ -426,7 +426,7 @@ Configure the styles.
 
 Execute the above code to render the following output.
 
-![](/js/SplitButton/Easy-customization_images/Easy-customization_img3.png) 
+![Content Type in JavaScript Split Button](/js/SplitButton/Easy-customization_images/Easy-customization_img3.png) 
```

Comment:
the second word should start in lower case

Reply:
Updated